concordium-consensus/src/Concordium/External.hs

```diff
@@ -1135,17 +1135,22 @@ checkIfWeAreFinalizer cptr = do
 -- Returns -1 if we are not added as a baker.
 -- Returns -2 if we are added as a baker, but not part of the baking committee yet.
 -- Returns -3 if we have keys that do not match the baker's public keys on the chain.
+-- Returns -5 if baking has stopped due to consensus being shut down.
```

Comment:
Why -5 rather than -4?

Reply:
Yes.. good question.. I think  I had a talk with @abizjak about the solution to this, where -5 was mentioned, and I never gave it extra thought. I'll change it.

Reply:
Don't blame me. `-5` was representative of "numbers not yet in use".

Reply:
> Don't blame me. `-5` was representative of "numbers not yet in use".

Can't argue with that.